### ISSUE TITLE
Toggling show boosts on profile not working

### ISSUE THREAD

**Issue by obsoleszenz** [cheeaun/phanpy#494]
**Describe the bug**
- Which site: phanpy.social
- Which site version: 2024.04.04.65d51b0
- Which instance: nerdculture.de

**To Reproduce**
1. Go to a profile
2. Try enabling show boosts
3. boosts still not shown and icon being prefixed with a minus
v
**Expected behavior**
Being able to enable showing boosts

**Screenshots**
[Screencast_20240413_154019.webm](https://github.com/cheeaun/phanpy/assets/94946669/579bfca9-74e1-426c-9a25-fa2159a16168)

**Desktop (please complete the following information):**
 - OS: linux
 - Browser: firefox
 - Version: Mozilla Firefox 124.0

**Additional context**

**Comment by cheeaun** [comment]
@obsoleszenz without clicking on '- Boosts', if you scroll down, do you see the boosts carousel?

**Comment by obsoleszenz** [comment]
> @obsoleszenz without clicking on '- Boosts', if you scroll down, do you see the boosts carousel?

yes that is working. Seems like only the toggle indicator is not working. Actually same with replies

**Comment by cheeaun** [comment]
@obsoleszenz '- Boosts' actually means **exclude** Boosts from the list of posts.

**Comment by obsoleszenz** [comment]
I would have expected the - to turn into a + depending on the toggle state?
